common/src/main/java/dev/schmarrn/lighty/core/LightyRenderer.java

```diff
@@ -81,20 +81,23 @@ private static int addData(SectionPos chunkSection, SectionBuffers gpuBuffer, Ve
 
     private static int goThroughEachBuffer(Minecraft minecraft, Camera camera, Vec3 camPos, Frustum frustum, List<RenderPass.Draw<GpuBufferSlice[]>> drawList, List<DynamicUniforms.Transform> transforms) {
         int biggestBufferSize = 0;
-        for (var entry : Compute.cachedBuffers.entrySet()) {
+        for (var cacheIterator = Compute.cachedBuffers.object2ObjectEntrySet().fastIterator(); cacheIterator.hasNext();) {
+            var entry = cacheIterator.next();
             SectionPos chunkSection = entry.getKey();
             BufferHolder cachedBuffer = entry.getValue();
-
-            for (var bufferEntry : cachedBuffer.getGpuBuffers().entrySet()) {
-                String key = bufferEntry.getKey();
-                if (!cachedBuffer.isValid(key)) {
+            var gpuBuffers = cachedBuffer.getGpuBuffers();
+            for (var bufferIterator = gpuBuffers.object2ObjectEntrySet().fastIterator(); bufferIterator.hasNext();) {
+                var bufferEntry = bufferIterator.next();
+                if (!cachedBuffer.isValid(bufferEntry.getKey())) {
                     continue;
                 }
-                if (!frustum.isVisible(
-                        AABB.encapsulatingFullBlocks(chunkSection.origin().offset(-1, -1, -1), chunkSection.origin().offset(16, 16, 16))
-                )) {
+
+                var sectionOrigin = chunkSection.origin();
+                var chunkBoundaries = AABB.encapsulatingFullBlocks(sectionOrigin.offset(-1, -1, -1), sectionOrigin.offset(16, 16, 16));
+                if (!frustum.isVisible(chunkBoundaries)) {
```

Comment:
Question: Is there a benefit over the previous version, other than the shorter line length?